Makefile

```diff
@@ -187,6 +187,16 @@ endif
 .PHONY: podman
 podman: bin/podman
 
+.PHONY: bin/podman-remote-static
+podman-remote-static: bin/podman-remote-static
+	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -f Containerfile.static -t podmanbuildimage .
+	${CONTAINER_RUNTIME} run --rm --security-opt label=disable -v $$(pwd):/src/github.com/containers/libpod \
+		podmanbuildimage make podman-remote-static-local
+
+.PHONY: bin/podman-remote-static-local
+podman-remote-static-local: bin/podman-remote-static
+	CGO_ENABLED=0 $(GO) build $(BUILDFLAGS) -gcflags '$(GCFLAGS)' -asmflags '$(ASMFLAGS)' -ldflags '$(LDFLAGS_PODMAN_STATIC)' -tags "!ABISupport varlink containers_image_openpgp remoteclient" -o bin/podman-remote-static $(PROJECT)/cmd/podman
+
```

Comment:
Do we need a `install.podman-remote-static-local` target too?

Reply:
No we do not want this to be installed, since we would prefer people to use dynamically linked podman-remote.  The only reason to build this is for people who want to use podman-remote on distros without compiling and packaging.  Mainly doing this for CRC.